test/unit/test_bigquery_adapter.py

```diff
@@ -683,6 +789,7 @@ def test_parse_partition_by(self):
             }).to_dict(), {
                 "field": "id",
                 "data_type": "int64",
+                "granularity": "DAY",
```

Comment:
Sorry, missed this one:
```suggestion
                "granularity": "day",
```